apps/mobile/src/libs/components/screen-wrapper/screen-wrapper.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import {
+	SafeAreaView,
+	useSafeAreaInsets,
+} from "react-native-safe-area-context";
+
+type Props = {
+	children: React.ReactNode;
+};
+
+const ScreenWrapper: React.FC<Props> = ({ children }) => {
```

Comment:
I meant passing the `edges` prop to the ScreenWrapper itself to be more flexible. For instance, in some cases, you'll need only a safe area at the top, not at the bottom, or vice versa.

Reply:
Working on this

Reply:
Add edges props, create enum for edges and using this component in the 'auth' screen